### PR TITLE
Update CPAT to 1.2.4

### PR DESCRIPTION
**Pull request by rhpvorderman**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Cpat is installable from Pypi now. But this does not install the r-dependency. So long live conda! Also this will make a biocontainer for easy usage. 

Build the recipe using conda skeleton and updated the existing one accordingly.

### CODE REVIEW

**Comment by rhpvorderman** [comment]
@bioconda/core 
Technically this is not a new package, neither is it my first package. But this package was made by somebody else, I completely overhauled it _and_ it is my first python package on conda.
I would feel a lot better if this was reviewed before the merge button is pressed. Thanks!

**Comment by dpryan79** [comment]
Can you try enabling building on python 3? I think the most recent version of bx-python supports it finally.

**Comment by rhpvorderman** [comment]
> Can you try enabling building on python 3? I think the most recent version of bx-python supports it finally.

No. Because CPAT is written entirely in 2.7. (Yes it sucks, but luckily we can run it in a container).

File: recipes/cpat/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
 package:
-  name: cpat
-  version: 1.2.3
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
 
 build:
-  number: 1
-  skip: True # [not py27] # because of bx-python
+  noarch: true
```
Comment (dpryan79) [inline recipes/cpat/meta.yaml]: You presumably mean `noarch: python`

Comment (rhpvorderman) [reply recipes/cpat/meta.yaml]: Thanks!

Comment (rhpvorderman) [reply recipes/cpat/meta.yaml]: @dpryan79 it is done. Thanks again.

**Comment by dpryan79** [comment]
Nice job @rhpvorderman !

**Pull request merged**